app/api/server/v1/users.js

```diff
@@ -132,6 +132,31 @@ API.v1.addRoute('users.setActiveStatus', { authRequired: true }, {
 	},
 });
 
+API.v1.addRoute('users.deactivateIdle', { authRequired: true }, {
+	post() {
+		console.log(this.bodyParams);
```

Comment:
I believe this is a left-over from dev time

Reply:
Thanks